apollo-rx-support/src/main/java/com/apollographql/apollo/rx/RxApollo.java

```diff
@@ -170,6 +171,33 @@ private RxApollo() {
     }, backpressureMode);
   }
 
+  /**
+   * Converts an {@link ApolloStoreOperation} to a Observable.
+   *
+   * @param operation        the ApolloStoreOperation to convert
+   * @param <T>              the value type
+   * @param backpressureMode The {@link rx.Emitter.BackpressureMode} to use.
+   * @return the converted Observable
+   */
+  @NotNull public static <T> Observable<T> from(@NotNull final ApolloStoreOperation<T> operation,
```

Comment:
can a store operation return more than 1 result?

Reply:
https://github.com/apollographql/apollo-android/blob/master/apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/ApolloStoreOperation.java#L82

Seems to only call `onSuccess` once for 1 result

Reply:
I think we can get away with a rx single with no Backpressure